optuna/cli.py

```diff
@@ -50,6 +48,174 @@ def _check_storage_url(storage_url: Optional[str]) -> str:
     return storage_url
 
 
+def _format_value(value: Any) -> Any:
+    #  Format value that can be serialized to JSON or YAML
```

Comment:
Please add a period at the end of the sentence (See [Coding Style Conventions](https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#block-and-inline-comments-are-supposed-to-be-sentence-s-)).
```suggestion
    #  Format value that can be serialized to JSON or YAML.
```